client/app/components/auth/signin.html

```diff
@@ -1,14 +1,14 @@
 <div class="authDiv">
-  <h1>Signin</h1>
+  <h1>Sign In</h1>
     Username:
     <br>
-    <input type="text" ng-model="user.username">
+    <input type="text" ng-model="userCredentials.username">
     <br>
     Password:
     <br>
-    <input type="text" ng-model="user.password">
+    <input type="text" ng-model="userCredentials.password">
     <br>
-    <button ng-click="signin()">Signin</button>
+    <button ng-click="signIn(userCredentials)">SignIn</button>
```

Comment:
This `signIn` function doesn't take an argument. It uses `$scope.userCredentials` internally.